### DISCUSSION TITLE
Input parameters are not passed to docker entrypoint when action.yaml has .yaml extension

### DISCUSSION THREAD

**Discussion by lucianposton** [community/community#26192] [Actions]
<p>When creating an action using a docker image, if your metadata file is named <code>action.yaml</code>, the action appears to work fine in all regards, except that input parameters are not passed to <code>/usr/bin/docker</code> as seen below, causing the entrypoint to not receive any arguments.</p>
<p><img src="https://user-images.githubusercontent.com/646121/181096006-abe85d1b-d294-483d-a4f7-f5f719b1b29c.png" alt="lol2.png" data-base62-sha1="2yCjA0HWxwSw2GbQrf195A62twP" width="658" height="160"></p>
<p>If instead the action’s metadata file is named <code>action.yml</code>, the parameters will be passed as expected. In the image below, notice how there is a parameter, “bob”, passed to /usr/bin/docker.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/7625197ff944af7c36fac26e061869b2166bf4e1.png" data-download-href="/uploads/short-url/gR9PhdedX6PP7TS07ZvIN9IefSx.png?dl=1" title="lol3.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/646121/181096007-c08bbfe1-7045-4b3e-b507-4170d6e93371.png" alt="lol3.png" data-base62-sha1="gR9PhdedX6PP7TS07ZvIN9IefSx" width="690" height="181" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/7625197ff944af7c36fac26e061869b2166bf4e1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">lol3.png</span><span class="informations">747×196 28 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Repro:</p>
<ol>
<li>
<p>Fork <a href="https://github.com/actions/hello-world-docker-action" rel="nofollow noopener">https://github.com/actions/hello-world-docker-action</a></p>
</li>
<li>
<p>In your fork, <code>git mv action.yml action.yaml</code> and push the commit</p>
</li>
<li>
<p>Follow <a href="https://help.github.com/en/actions/building-actions/creating-a-docker-container-action#testing-out-your-action-in-a-workflow" rel="nofollow noopener">https://help.github.com/en/actions/building-actions/creating-a-docker-container-action#testing-out-your-action-in-a-workflow</a> and set the using action to your fork <span class="mention">@master</span>.</p>
</li>
</ol>
<p>Similar to <a href="https://github.community/t/actions-input-parameters-are-not-passed-to-images-pulled-from-a-docker-repository/16473">https://github.community/t5/GitHub-Actions/Actions-input-parameters-are-not-passed-to-images-pulled-from-a/td-p/33883</a></p>

**Comment by Yanjingzhu** [comment]
<p>Thank you for pointing this out. I have directed your feedback to the appropriate engineering team for further evaluation. When they give me any feedback, I will update here at once. </p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p><a class="mention" href="/lucianposton">@lucianposton</a> The <a href="https://github.com/actions/runner/commit/c59c0e2ded040f151ccd8845856def6b4cd7b58a" rel="nofollow noopener">change </a>to support action.yaml is part of 2.165.x runner release which hasn’t released to all production Rings. We have scheduled to release to the last Ring tomorrow. Please wait for a few days for the changes can be used. Thank you for your understanding.  </p>

**Comment by Yanjingzhu** [comment]
<p><a class="mention" href="/lucianposton">@lucianposton</a>  The <a href="https://github.com/actions/runner/commit/c59c0e2ded040f151ccd8845856def6b4cd7b58a" rel="nofollow noopener">change </a>to support action.yaml has released to all production Rings. I tested in my side, the action.yaml could work now. How about yours? </p>

**Comment by lucianposton** [comment]
<p>The issue seems to be resolved. I was unable to reproduce the problem.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9a261fc8351fe94270b4102c702843ad6a913bb1.png" data-download-href="/uploads/short-url/lZFd5jJZ2vklg3HDgLEeBUq80kV.png?dl=1" title="screenshot-2020-02-26T16:10:57.057Z.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/646121/181096013-2870b4d3-5f9d-4632-92f9-0206a106fe1a.png" alt="screenshot-2020-02-26T16:10:57.057Z.png" data-base62-sha1="lZFd5jJZ2vklg3HDgLEeBUq80kV" width="690" height="210" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9a261fc8351fe94270b4102c702843ad6a913bb1_2_690x210.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/9a261fc8351fe94270b4102c702843ad6a913bb1_2_1035x315.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/9/9a261fc8351fe94270b4102c702843ad6a913bb1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9a261fc8351fe94270b4102c702843ad6a913bb1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">screenshot-2020-02-26T16:10:57.057Z.png</span><span class="informations">1116×341 46.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks</p>
